docs/production/self-hosting/hosting-with-docker.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 69
+title: Docker
+---
+
+
+# Deploying with Docker
+
+This guide shows you how to setup a starter DBOS Python application and its Postgres database using Docker.
+
+## Dockerfile
+
+First, we'll setup a Dockerfile to configure a Debian Python image.
+
+DBOS is just a library for your program to import, so the only core requirement is to run `pip install dbos`.
```

Comment:
```suggestion
There's nothing unique about this Dockerfile&mdash;DBOS is just a library for your program to import, so it can run in any Docker container that has the appropriate language runtime (Node or Python) installed.
```